diff --git a/src/Http/Resources/CP/Entries/Entries.php b/src/Http/Resources/CP/Entries/Entries.php
index 8813894f0..7ab4d90b1 100644
--- a/src/Http/Resources/CP/Entries/Entries.php
+++ b/src/Http/Resources/CP/Entries/Entries.php
@@ -44,9 +44,7 @@ class Entries extends ResourceCollection
 
         return [
             'data' => $this->collection->each(function ($entry) {
-                $entry
-                    ->blueprint($this->blueprint)
-                    ->columns($this->requestedColumns());
+                $entry->columns($this->requestedColumns());
             }),
 
             'meta' => [
diff --git a/src/Http/Resources/CP/Entries/ListedEntry.php b/src/Http/Resources/CP/Entries/ListedEntry.php
index 145c25462..7ea39e27b 100644
--- a/src/Http/Resources/CP/Entries/ListedEntry.php
+++ b/src/Http/Resources/CP/Entries/ListedEntry.php
@@ -8,16 +8,8 @@ use Statamic\Facades\User;
 
 class ListedEntry extends JsonResource
 {
-    protected $blueprint;
     protected $columns;
 
-    public function blueprint($blueprint)
-    {
-        $this->blueprint = $blueprint;
-
-        return $this;
-    }
-
     public function columns($columns)
     {
         $this->columns = $columns;
@@ -53,7 +45,7 @@ class ListedEntry extends JsonResource
         return $this->columns->mapWithKeys(function ($column) use ($extra) {
             $key = $column->field;
 
-            $value = $this->blueprint
+            $value = $this->resource->blueprint() // blinked
                 ->field($key)
                 ->setValue($extra[$key] ?? $this->resource->value($key))
                 ->setParent($this->resource)
